### PR TITLE
Deprecate contrast level

### PR DESCRIPTION
**Pull request by iapetus98**

### CODE REVIEW

File: print-api/src/main/java/com/aevi/print/model/ImageRow.java [hunk 1]
```diff
@@ -108,7 +110,9 @@ public int getContrastLevel() {
      *
      * @param contrastLevel the contrastLevel to set
      * @return This ImageRow object
+     * @deprecated Deprecated since 1.1.5, has no effect
```
Comment (louis-barman) [inline print-api/src/main/java/com/aevi/print/model/ImageRow.java]: may be the wording could be "has no effect. Ensure all contrast adjustments are made on the original image"

**Comment by louis-barman** [comment]
Please remove `contrastLevel` from the sample code as well.

**Comment by louis-barman** [comment]
Also the release notes need updating!

File: print-api/src/main/java/com/aevi/print/model/ImageRow.java [hunk 2]
```diff
@@ -94,8 +94,10 @@ public ImageRow align(Alignment alignment) {
      * Gets the contrast level of this image row. The contrast level is a
      * percentage value between 0 and 100, where 100 is full contrast.
      *
-     * @return the contrastLevel to set,
+     * @return the contrastLevel to set
+     * @deprecated Deprecated since 1.1.5, has no effect
```
Comment (louis-barman) [inline print-api/src/main/java/com/aevi/print/model/ImageRow.java]: We don't need the version number i think that should be in the release notes.

Comment (iapetus98) [reply print-api/src/main/java/com/aevi/print/model/ImageRow.java]: Recommended use of the deprecated tag is to specify the version number where it's deprecated from - we're looking at the Javadoc of an API which will be available in-IDE unlike the release notes...

**Pull request merged**